docs/src/main/paradox/docs/delta/api/current/kg-schemas-api.md

```diff
@@ -228,7 +236,39 @@ where...
 **Example**
```

Comment:
The listings query parameters should be put in a multiline, so that it is more readable, as done in organizations docs

Reply:
Also the `{sort}` field should say that by default the sorting is done by `_createdBy` and `@id`